app/models/concerns/smooch_search.rb

```diff
@@ -160,7 +161,8 @@ def search_for_similar_published_fact_checks(type, query, team_ids, after = nil,
 
     # "type" is text, video, audio or image
     # "query" is either a piece of text of a media URL
-    def search_for_similar_published_fact_checks_no_cache(type, query, team_ids, after = nil, feed_id = nil, language = nil)
+    def search_for_similar_published_fact_checks_no_cache(type, query, team_ids, after = nil, feed_id = nil, language = nil, limit = nil)
+      limit ||= CheckConfig.get(:most_relevant_team_limit, 3, :integer)
```

Comment:
Like the other comments: I suggest we always enforce a `limit` to be passed.

Reply:
Done